src/browser/main.coffee

```diff
@@ -69,7 +69,9 @@ setupCrashReporter = ->
 setupAtomHome = ->
   return if process.env.ATOM_HOME
 
-  atomHome = path.join(app.getHomeDir(), '.atom')
+  atomHome = path.join(path.dirname(process.execPath), 'userdata')
```

Comment:
Well, I don't own a Mac to know this, but we're talking about a simple portable way for Atom, how it will be made using a Mac?

Clarifying, this only will work if you put an "userdata" folder (empty or a copy of ".atom" contents) inside the executable directory. In my case, I'm using a USB drive to use Atom on any computer I plug it.